### PR TITLE
Adds a stun chemical (uplink only)

### PR DESCRIPTION
**Pull request by areitpog**
## About the PR
Adds a stun chemical that, well, stuns you while it's in your system.

IMO, it has some cool pacifist traitor potential and works well when combined with a hypopen

Demo:
https://user-images.githubusercontent.com/77842470/159936267-04a5c051-1d78-4495-9410-41ae223609eb.mp4

**Changelog**
:cl: areit
- add: Added a stun chemical that you can get from the uplink.

### CODE REVIEW

File: Content.Server/Chemistry/ReagentEffects/Paralyze.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    /// <remarks>
+    ///     true - refresh paralyze time,  false - accumulate paralyze time
+    /// </remarks>
+    [DataField("refresh")] public bool Refresh = true;
+
+    public override void Effect(ReagentEffectArgs args)
+    {
+        EntitySystem.Get<StunSystem>().TryParalyze(args.SolutionEntity, TimeSpan.FromSeconds(ParalyzeTime), Refresh);
+
+        args.Source?.RemoveReagent(args.Reagent.ID, args.Quantity); // Remove reagent, so that, presumably, it doesn't reapply 317589179857 times(?)
```
Comment (mirrorcult) [inline Content.Server/Chemistry/ReagentEffects/Paralyze.cs:25]: do not do this, metabolism already handles this

Comment (mirrorcult) [reply Content.Server/Chemistry/ReagentEffects/Paralyze.cs:25]: unless the behavior you're looking for is "stuns once and then deletes itself and doesnt do it again" in which case, i dont really see why it should do that?

File: Resources/Prototypes/Recipes/Reactions/chemicals.yml [hunk 2]
```diff
@@ -284,3 +284,15 @@
...
+#  id: Nocturine
+#  reactants:
+#    Tricordrazine:
+#      amount: 1
+#    Nitrogen:
+#      amount: 1
+#    Hydrogen:
+#      amount: 1
+#  products:
+#    Nocturine: 1
```
Comment (mirrorcult) [inline Resources/Prototypes/Recipes/Reactions/chemicals.yml:298]: if its unused just dont add it

**Review by mirrorcult — CHANGES_REQUESTED** [review]
i approve the concept, just nitpicks

File: Content.Server/Chemistry/ReagentEffects/Paralyze.cs (hunk 1 above)
Comment (areitpog) [reply Content.Server/Chemistry/ReagentEffects/Paralyze.cs]: should be good now

**Comment by areitpog** [comment]
should be good now

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**